OPM build failed: ecfvstencil.hh:250:58: error: ‘ePtr’ was not declared

```
          ^
Linking CXX executable bin/art2dgf
[  0%] Built target art2dgf
Scanning dependencies of target co2injection_flash_ecfv
[  1%] Building CXX object CMakeFiles/co2injection_flash_ecfv.dir/tests/co2injection_flash_ecfv.cc.o
In file included from <https://ci.statoil.no/job/OPM7/ws/ewoms/ewoms/disc/ecfv/ecfvdiscretization.hh>:34:0,
                 from <https://ci.statoil.no/job/OPM7/ws/ewoms/tests/co2injection_flash_ecfv.cc>:37:
<https://ci.statoil.no/job/OPM7/ws/ewoms/ewoms/disc/ecfv/ecfvstencil.hh>: In member function ‘void Ewoms::EcfvStencil<Scalar, GridView>::updateTopology(const Element&)’:
<https://ci.statoil.no/job/OPM7/ws/ewoms/ewoms/disc/ecfv/ecfvstencil.hh>:250:58: error: ‘ePtr’ was not declared in this scope
         subControlVolumes_.emplace_back(SubControlVolume(ePtr));
                                                          ^
<https://ci.statoil.no/job/OPM7/ws/ewoms/ewoms/disc/ecfv/ecfvstencil.hh>: In member function ‘void Ewoms::EcfvStencil<Scalar, GridView>::updatePrimaryTopology(const Element&)’:
<https://ci.statoil.no/job/OPM7/ws/ewoms/ewoms/disc/ecfv/ecfvstencil.hh>:285:58: error: ‘ePtr’ was not declared in this scope
         subControlVolumes_.emplace_back(SubControlVolume(ePtr));
                                                          ^
make[2]: *** [CMakeFiles/co2injection_flash_ecfv.dir/tests/co2injection_flash_ecfv.cc.o] Error 1
make[1]: *** [CMakeFiles/co2injection_flash_ecfv.dir/all] Error 2
make: *** [all] Error 2
Build step 'Execute shell' marked build as failure
[WARNINGS] Skipping publisher since build result is FAILURE
```

--- a/ewoms/disc/ecfv/ecfvstencil.hh
+++ b/ewoms/disc/ecfv/ecfvstencil.hh
@@ -241,14 +241,14 @@ public:
 #if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 4)
         // add the "center" element of the stencil
         subControlVolumes_.clear();
-        subControlVolumes_.emplace_back(SubControlVolume(element));
+        subControlVolumes_.emplace_back(/*SubControlVolume(*/element/*)*/);
         elements_.clear();
         elements_.emplace_back(element);
 #else
         // add the "center" element of the stencil
         ElementPointer ePtr(element);
         subControlVolumes_.clear();
-        subControlVolumes_.emplace_back(SubControlVolume(ePtr));
+        subControlVolumes_.emplace_back(/*SubControlVolume(*/ePtr/*)*/);
         elements_.clear();
         elements_.emplace_back(ePtr);
 #endif
@@ -262,12 +262,12 @@ public:
             // degree of freedom and an internal face, else add a
             // boundary face
             if (intersection.neighbor()) {
-                elements_.emplace_back(intersection.outside());
-                subControlVolumes_.emplace_back(SubControlVolume(intersection.outside()));
-                interiorFaces_.emplace_back(SubControlVolumeFace(intersection, subControlVolumes_.size() - 1));
+                elements_.emplace_back(/*SubControlVolume(*/intersection.outside()/*)*/);
+                subControlVolumes_.emplace_back(/*SubControlVolume(*/intersection.outside()/*)*/);
+                interiorFaces_.emplace_back(/*SubControlVolumeFace(*/intersection, subControlVolumes_.size() - 1/*)*/);
             }
             else {
-                boundaryFaces_.emplace_back(SubControlVolumeFace(intersection, - 10000));
+                boundaryFaces_.emplace_back(/*SubControlVolumeFace(*/intersection, - 10000/*)*/);
             }
         }
     }
@@ -277,14 +277,14 @@ public:
 #if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 4)
         // add the "center" element of the stencil
         subControlVolumes_.clear();
-        subControlVolumes_.emplace_back(SubControlVolume(element));
+        subControlVolumes_.emplace_back(/*SubControlVolume(*/element/*)*/);
         elements_.clear();
         elements_.emplace_back(element);
 #else
         // add the "center" element of the stencil
         ElementPointer ePtr(element);
         subControlVolumes_.clear();
-        subControlVolumes_.emplace_back(SubControlVolume(ePtr));
+        subControlVolumes_.emplace_back(/*SubControlVolume(*/ePtr/*)*/);
         elements_.clear();
         elements_.emplace_back(ePtr);
 #endif

--- a/ewoms/disc/ecfv/ecfvstencil.hh
+++ b/ewoms/disc/ecfv/ecfvstencil.hh
@@ -246,10 +246,11 @@ public:
         elements_.emplace_back(element);
 #else
         // add the "center" element of the stencil
+        ElementPointer ePtr(element);
         subControlVolumes_.clear();
         subControlVolumes_.emplace_back(SubControlVolume(ePtr));
         elements_.clear();
-        elements_.emplace_back(ElementPointer(element));
+        elements_.emplace_back(ePtr);
 #endif
 
         interiorFaces_.clear();
@@ -281,10 +282,11 @@ public:
         elements_.emplace_back(element);
 #else
         // add the "center" element of the stencil
+        ElementPointer ePtr(element);
         subControlVolumes_.clear();
         subControlVolumes_.emplace_back(SubControlVolume(ePtr));
         elements_.clear();
-        elements_.emplace_back(ElementPointer(element));
+        elements_.emplace_back(ePtr);
 #endif
     }
 

